libs/database/src/lib/services/authentication.service.spec.ts

```diff
@@ -321,6 +321,46 @@ describe('Authentication Service', () => {
       });
     });
 
+    describe('#getExistedUserFromCastcleId', () => {
+      let signupResult: AccountActivation;
+      const signupRequirements: SignupRequirements = {
+        displayId: 'people',
+        displayName: 'People',
+        email: 'sompopdude@dudedude.com',
+        password: '2@HelloWorld',
+      };
+      beforeAll(async () => {
+        signupResult = await service.signupByEmail(
+          createAccountResult.accountDocument,
+          {
+            displayId: 'people',
+            displayName: 'People',
+            email: signupRequirements.email,
+            password: signupRequirements.password,
+          }
+        );
+      });
+      it('should create an accountActivation', () => {
+        expect(signupResult).toBeDefined();
+      });
+      it('should return exist user is null', async () => {
+        const id = 'undefined';
+        const findUser = await service.getExistedUserFromCastcleId(id);
+        expect(findUser).toBeNull();
+      });
+      it('should return exist user not null', async () => {
+        const id = 'people';
+        const findUser = await service.getExistedUserFromCastcleId(id);
+        expect(findUser).not.toBeNull();
+      });
+      it('should return null after delete user', async () => {
+        const id = 'people';
+        const findUser = await service.getExistedUserFromCastcleId(id);
+        findUser.visibility = EntityVisibility.Deleted;
+        expect(findUser.visibility).toEqual(EntityVisibility.Deleted);
```

Comment:
expected result ตรงนี้แนะนำให้ทำชัดเจนกว่านี้ได้ไหมครับ เช่น 'should return null after delete user' findUser จะต้องเป็น null หลังจาก delete user